pom.xml

```diff
@@ -195,6 +200,14 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.18.1</version>
+          <dependencies>
+            <dependency>
+              <!-- Force junit provider because surefire will choose the testng one if testng is detected -->
+              <groupId>org.apache.maven.surefire</groupId>
+              <artifactId>surefire-junit47</artifactId>
```

Comment:
@juherr 
Try to use both providers in this section.
Each of them makes a lookup of dedicated test. Since testng provider can run junit tests as well you can suppress this by setting junit=false, see it in our documentation but Version 2.18.1 is too old:
http://maven.apache.org/surefire/maven-surefire-plugin/examples/testng.html#Running_TestNG_and_JUnit_Tests